elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryCleanerThread.java

```diff
@@ -77,7 +78,8 @@ private String evaluateFormattedFilterDate(int calendarUnit, int amount) {
         cal.setTime(new Date());
         cal.add(calendarUnit, -(amount));
         Date filterDate = cal.getTime();
-        Format dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
+        ISO8601DateSerde serde = (ISO8601DateSerde) elide.getElideSettings().getSerdes().get(Date.class);
```

Comment:
This may not be a safe cast.  There are other Serde's for Date besides ISO8601.  Also, the client has control over this so let's figure out what can work in all circumstances.